app/Livewire/SettingsOauth.php

```
@@ -38,16 +47,48 @@ public function mount()
     private function updateOauthSettings(?string $provider = null)
     {
         if ($provider) {
-            $oauth = $this->oauth_settings_map[$provider];
+            $oauthData = $this->oauth_settings_map[$provider];
+            $oauth = OauthSetting::find($oauthData['id']);
```

PHPMD (2.15.0)

Avoid using static access to class '\App\Models\OauthSetting' in method 'updateOauthSettings'. (undefined)

(StaticAccess)